app/vmselect/promql/exec_test.go

```diff
@@ -9488,6 +9488,17 @@ func TestNoImplicitConversionRequiredTrue(t *testing.T) {
 	f("max_over_time(rate(my_counter_total[5m])[1h:1m])[5m:1m]")
 	f("max_over_time(rate(my_counter_total[5m])[1h:])[5m:]")
 
+	// this query will fail the test because it contains subquery
+	// f(`aggr_over_time(("rate", "increase"), foo[1h])`)
```

Comment:
I don't get this. From my understanding, `aggr_over_time(("rate", "increase"), foo)` is ImplicitConversionNeeded, but foo[1h] is a complete subquery which is allowed.